backend/src/zac/werkvoorraad/serializers.py

```diff
@@ -0,0 +1,132 @@
+from django.core.validators import URLValidator
+from django.urls import reverse
+from django.utils.translation import gettext as _
+
+from rest_framework import serializers
+from zgw_consumers.api_models.catalogi import ZaakType
+from zgw_consumers.drf.serializers import APIModelSerializer
+
+from zac.accounts.api.serializers import ZaakAccessSerializer
+from zac.accounts.models import AccessRequest, User
+from zac.activities.api.serializers import ActivitySerializer
+from zac.activities.models import Activity
+from zac.camunda.data import Task
+from zac.core.api.serializers import ZaakSerializer
+from zgw.models.zrc import Zaak
+
+from .data import AccessRequestGroup, ActivityGroup
+
+
+class AccessRequesterSerializer(APIModelSerializer):
+    requester = serializers.SlugRelatedField(
+        slug_field="username",
+        queryset=User.objects.all(),
+        help_text=_("Username of access requester/grantee"),
+    )
+
+    class Meta:
+        model = AccessRequest
+        fields = ("requester",)
+
+
+class WorkStackAccessRequestsSerializer(APIModelSerializer):
+    requesters = AccessRequesterSerializer(many=True)
```

Comment:
```suggestion
    access_requests = AccessRequestSerializer(many=True)
```

nit-picky, but "access requester" makes me think about just the user behind the access request, not the access request itself. Also keep in mind if we add attributes to the access request and have to expose them in the API in the future, it would look semantically incorrect to have this keyed under `requesters`.